main.js

```diff
@@ -1,39 +1,47 @@
-let easyBoard = 10;
-let mediumBoard = 15;
-let extremeBoard = 20;
+let boardSize = 10;
```

Comment:
You can delete this variable all together and use directly constants values, like: 10 * 60 => 600;